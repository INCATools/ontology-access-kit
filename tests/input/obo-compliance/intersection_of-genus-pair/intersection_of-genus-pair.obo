format-version: 1.4
ontology: intersection_of-genus-pair

[Term]
id: X:1
intersection_of: X:2
intersection_of: X:3

