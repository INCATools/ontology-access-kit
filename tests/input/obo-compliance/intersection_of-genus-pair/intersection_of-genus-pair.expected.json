{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/intersection_of-genus-pair.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ],
    "logicalDefinitionAxioms" : [ {
      "definedClassId" : "http://purl.obolibrary.org/obo/X_1",
      "genusIds" : [ "http://purl.obolibrary.org/obo/X_2", "http://purl.obolibrary.org/obo/X_3" ]
    } ]
  } ]
}